mmv1/third_party/terraform/utils/config.go.erb

```diff
@@ -6,14 +6,18 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"net/url"
 	"regexp"
 	"strings"
 	"time"
 
-	"google.golang.org/api/option"
 	"github.com/hashicorp/go-cleanhttp"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/logging"
+	"google.golang.org/api/option"
 
+	kms "cloud.google.com/go/kms/apiv1"
+	dcl "github.com/GoogleCloudPlatform/declarative-resource-client-library/dcl"
+	eventarcDcl "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/eventarc/beta"
```

Comment:
Why were these pulled in? They are already listed below wrapped in 
```
<% unless version == 'ga' -%>
```

Reply:
Ah, yeah, that was unintentional. Thanks!

Reply:
removed in abf157c99b7e6732b8f1a96a12f6ce877a7e196e